src/schema/posts.ts

```diff
@@ -1400,19 +1384,26 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
   Query: {
     sourcePostModeration: async (
       _,
-      { id, sourceId }: { id: string; sourceId: string },
+      { id }: { id: string },
```

Comment:
The ID of the entity is a unique identifier in itself; we don't need to check for the `sourceId`.